mitosheet/src/components/taskpanes/PivotTable/InvalidSelectedColumnsError.tsx

```diff
@@ -45,11 +43,11 @@ const PivotInvalidSelectedColumnsError = (props: {
                 // Note: we hide X overflow so that really long column headers don't make the pivot
                 // scrollable
                 <div className='text-color-error' style={{overflowX: 'hidden'}}>
-                    The {invalidSelectedColumnIDs.length === 1 ? 'column' : 'columns'} {invalidSelectedColumnIDs.length === 1 ? invalidSelectedColumnIDs[0] : invalidSelectedColumnIDs.join(', ')} {invalidSelectedColumnIDs.length === 1 ? 'does' : 'do'} not exist in this sheet anymore. Delete {invalidSelectedColumnIDs.length === 1 ? 'it' : 'them'} to make this pivot valid.
+                    The {invalidSelectedColumnIDs.length === 1 ? 'column' : 'columns'} {invalidSelectedColumnIDs.length === 1 ? invalidSelectedColumnIDs[0] : invalidSelectedColumnIDs.join(', ')} {invalidSelectedColumnIDs.length === 1 ? 'does' : 'do'} not exist in this sheet anymore. Delete {invalidSelectedColumnIDs.length === 1 ? 'it' : 'them'} to create a valid configuration.
```

Comment:
This is a wild line of code. Can we clean this up? Maybe just saving the length variable, or saving a variable `isSingular` and using that in all these cases.